docs/generated/packages/react.json

```diff
@@ -717,6 +717,10 @@
             "type": "boolean",
             "description": "Generate JavaScript files rather than TypeScript files.",
             "default": false
+          },
+          "ignorePaths": {
```

Comment:
One consideration here is that all of our other options  use path relative to the workspace root, not the project root.

We should do that here as well for consistency.